### PR TITLE
Update ca.m3u

### PR DESCRIPTION
**Pull request by tpkahlon**
- Add stream in this category

### CODE REVIEW

File: channels/ca.m3u [hunk 1]
```diff
@@ -127,6 +127,10 @@ http://159.69.58.154/cpmtv/cpmtv.m3u8
 https://pe-fa-lp02a.9c9media.com/live/News1Digi/p/hls/00000201/38ef78f479b07aa0/index/0c6a10a2/live/stream/h264/v1/3500000/manifest.m3u8
 #EXTINF:-1 tvg-id="CTVToronto.ca" tvg-country="CA" tvg-language="English" tvg-logo="https://d229kpbsb5jevy.cloudfront.net/teleup/content/common/logos/channel/logos/uafvsq.png" group-title="",CTV Toronto (720p)
 https://bozztv.com/teleyupp1/teleup-zxsJFt6VvY/playlist.m3u8
+#EXTINF:-1 tvg-id="Dashmesh Culture Centre Calgary" tvg-country="CA" tvg-language="Punjabi" tvg-logo="https://yt3.ggpht.com/ytc/AKedOLTnO6e7pkvYiO2q7Lh_sWIk7RYtvlf1yXpEnySw=s48-c-k-c0x00ffffff-no-rj" group-title="",Dashmesh Culture Centre Calgary
+https://youtu.be/hHVlXaBxL-o
```
Comment (freearhey) [inline channels/ca.m3u:131]: YouTube links are not supported.

File: channels/ca.m3u [hunk 2]
```diff
@@ -210,6 +216,10 @@ http://primeasia.dyndns.tv:8080/Live_web_250/index.m3u8
 http://cdn27.live247stream.com/primecanada/247/primecanada/stream1/playlist.m3u8
 #EXTINF:-1 tvg-id="QVTV.ca" tvg-country="CA" tvg-language="English" tvg-logo="https://i.imgur.com/X9LBFzK.jpg" group-title="Religious",QVTV (720p)
 https://qvmstream.tulix.tv/720p/720p/playlist.m3u8
+#EXTINF:-1 tvg-id="Rexdale Gurdwara Sahib" tvg-country="CA" tvg-language="Punjabi" tvg-logo="https://www.sikhspiritualcentrerexdale.com/wp-content/uploads/2019/07/favicon-transprent.png" group-title="",Rexdale Gurdwara Sahib
+https://cdn12.henico.net:8443/live/ssct/index.m3u8
+#EXTINF:-1 tvg-id="Sahib Sukh Sagar Canada" tvg-country="CA" tvg-language="Punjabi" tvg-logo="https://yt3.ggpht.com/ytc/AKedOLQC-nL2HpaRQboMz4LLmouIrcIuWFqaTubc325A=s48-c-k-c0x00ffffff-no-rj" group-title="",Sahib Sukh Sagar Canada
+https://youtu.be/36WffJQ8T4o
```
Comment (freearhey) [inline channels/ca.m3u:222]: YouTube links are not supported.

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/ca.m3u [hunk 3]
```diff
@@ -127,6 +127,10 @@ http://159.69.58.154/cpmtv/cpmtv.m3u8
 https://pe-fa-lp02a.9c9media.com/live/News1Digi/p/hls/00000201/38ef78f479b07aa0/index/0c6a10a2/live/stream/h264/v1/3500000/manifest.m3u8
 #EXTINF:-1 tvg-id="CTVToronto.ca" tvg-country="CA" tvg-language="English" tvg-logo="https://d229kpbsb5jevy.cloudfront.net/teleup/content/common/logos/channel/logos/uafvsq.png" group-title="",CTV Toronto (720p)
 https://bozztv.com/teleyupp1/teleup-zxsJFt6VvY/playlist.m3u8
+#EXTINF:-1 tvg-id="Dashmesh Culture Centre Calgary" tvg-country="CA" tvg-language="Punjabi" tvg-logo="https://yt3.ggpht.com/ytc/AKedOLTnO6e7pkvYiO2q7Lh_sWIk7RYtvlf1yXpEnySw=s48-c-k-c0x00ffffff-no-rj" group-title="",Dashmesh Culture Centre Calgary
+https://youtu.be/hHVlXaBxL-o
+#EXTINF:-1 tvg-id="Dukh Nivaran Sahib Surrey" tvg-country="CA" tvg-language="Punjabi" tvg-logo="https://www.dukhnivaran.ca/wp-content/uploads/2019/01/dukhnivaran_logo1-1.png" group-title="",Dukh Nivaran Sahib Surrey
```
Comment (freearhey) [inline channels/ca.m3u:132]: - tvg-id is wrong

File: channels/ca.m3u [hunk 4]
```diff
@@ -165,6 +169,8 @@ https://live.corusdigitaldev.com/groupd/live/deb8cec5-87fb-460a-ab82-0929374fc5f
 https://live.corusdigitaldev.com/groupb/live/564df695-94f9-4f27-b1b9-0a936ab01721/live.isml/.m3u8
 #EXTINF:-1 tvg-id="GurSikhSabhaTV.ca" tvg-country="CA" tvg-language="Hindi" tvg-logo="https://i.imgur.com/2EKCtQm.png" group-title="Religious",GurSikh Sabha TV (720p)
 http://cdn12.henico.net:8080/live/gsctv/index.m3u8
+#EXTINF:-1 tvg-id="Gurudwara Sahib Brookside" tvg-country="CA" tvg-language="Punjabi" tvg-logo="http://ramgarhiabc.com/wp-content/uploads/2014/02/headerlogo.png" group-title="",Gurudwara Sahib Brookside
```
Comment (freearhey) [inline channels/ca.m3u:172]: tvg-id is wrong

File: channels/ca.m3u [hunk 5]
```diff
@@ -210,6 +216,10 @@ http://primeasia.dyndns.tv:8080/Live_web_250/index.m3u8
 http://cdn27.live247stream.com/primecanada/247/primecanada/stream1/playlist.m3u8
 #EXTINF:-1 tvg-id="QVTV.ca" tvg-country="CA" tvg-language="English" tvg-logo="https://i.imgur.com/X9LBFzK.jpg" group-title="Religious",QVTV (720p)
 https://qvmstream.tulix.tv/720p/720p/playlist.m3u8
+#EXTINF:-1 tvg-id="Rexdale Gurdwara Sahib" tvg-country="CA" tvg-language="Punjabi" tvg-logo="https://www.sikhspiritualcentrerexdale.com/wp-content/uploads/2019/07/favicon-transprent.png" group-title="",Rexdale Gurdwara Sahib
```
Comment (freearhey) [inline channels/ca.m3u:219]: tvg-id is wrong

File: channels/ca.m3u [hunk 6]
```diff
@@ -210,6 +216,10 @@ http://primeasia.dyndns.tv:8080/Live_web_250/index.m3u8
...
+https://cdn12.henico.net:8443/live/ssct/index.m3u8
+#EXTINF:-1 tvg-id="Sahib Sukh Sagar Canada" tvg-country="CA" tvg-language="Punjabi" tvg-logo="https://yt3.ggpht.com/ytc/AKedOLQC-nL2HpaRQboMz4LLmouIrcIuWFqaTubc325A=s48-c-k-c0x00ffffff-no-rj" group-title="",Sahib Sukh Sagar Canada
```
Comment (freearhey) [inline channels/ca.m3u:221]: tvg-id is wrong

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/ca.m3u (hunk 1 above)
Comment (tpkahlon) [reply channels/ca.m3u]: @freearhey  Fixed!

File: channels/ca.m3u (hunk 2 above)
Comment (tpkahlon) [reply channels/ca.m3u]: @freearhey Fixed!

File: channels/ca.m3u (hunk 3 above)
Comment (tpkahlon) [reply channels/ca.m3u:130]: @freearhey Fixed!

File: channels/ca.m3u (hunk 4 above)
Comment (tpkahlon) [reply channels/ca.m3u]: @freearhey Fixed!

File: channels/ca.m3u (hunk 5 above)
Comment (tpkahlon) [reply channels/ca.m3u]: @freearhey Fixed!

File: channels/ca.m3u (hunk 6 above)
Comment (tpkahlon) [reply channels/ca.m3u]: @freearhey Fixed!

File: channels/ca.m3u (hunk 3 above)
Comment (freearhey) [reply channels/ca.m3u:130]: @tpkahlon please check again

Comment (tpkahlon) [reply channels/ca.m3u]: @freearhey Apologies, fixed!

File: channels/ca.m3u [hunk 7]
```diff
@@ -127,6 +127,8 @@ http://159.69.58.154/cpmtv/cpmtv.m3u8
 https://pe-fa-lp02a.9c9media.com/live/News1Digi/p/hls/00000201/38ef78f479b07aa0/index/0c6a10a2/live/stream/h264/v1/3500000/manifest.m3u8
 #EXTINF:-1 tvg-id="CTVToronto.ca" tvg-country="CA" tvg-language="English" tvg-logo="https://d229kpbsb5jevy.cloudfront.net/teleup/content/common/logos/channel/logos/uafvsq.png" group-title="",CTV Toronto (720p)
 https://bozztv.com/teleyupp1/teleup-zxsJFt6VvY/playlist.m3u8
+#EXTINF:-1 tvg-id="DukhNivaran.ca" tvg-country="CA" tvg-language="Punjabi" tvg-logo="https://www.dukhnivaran.ca/wp-content/uploads/2019/01/dukhnivaran_logo1-1.png" group-title="",Dukh Nivaran Sahib Surrey
```
Comment (freearhey) [inline channels/ca.m3u:130]:
```suggestion
#EXTINF:-1 tvg-id="DukhNivaranSahibSurrey.ca" tvg-country="CA" tvg-language="Punjabi" tvg-logo="https://www.dukhnivaran.ca/wp-content/uploads/2019/01/dukhnivaran_logo1-1.png" group-title="",Dukh Nivaran Sahib Surrey
```

File: channels/ca.m3u [hunk 8]
```diff
@@ -165,6 +167,8 @@ https://live.corusdigitaldev.com/groupd/live/deb8cec5-87fb-460a-ab82-0929374fc5f
 https://live.corusdigitaldev.com/groupb/live/564df695-94f9-4f27-b1b9-0a936ab01721/live.isml/.m3u8
 #EXTINF:-1 tvg-id="GurSikhSabhaTV.ca" tvg-country="CA" tvg-language="Hindi" tvg-logo="https://i.imgur.com/2EKCtQm.png" group-title="Religious",GurSikh Sabha TV (720p)
 http://cdn12.henico.net:8080/live/gsctv/index.m3u8
+#EXTINF:-1 tvg-id="RamgarhiABC.com" tvg-country="CA" tvg-language="Punjabi" tvg-logo="http://ramgarhiabc.com/wp-content/uploads/2014/02/headerlogo.png" group-title="",Gurudwara Sahib Brookside
```
Comment (freearhey) [inline channels/ca.m3u:170]:
```suggestion
#EXTINF:-1 tvg-id="GurudwaraSahibBrookside.ca" tvg-country="CA" tvg-language="Punjabi" tvg-logo="http://ramgarhiabc.com/wp-content/uploads/2014/02/headerlogo.png" group-title="",Gurudwara Sahib Brookside
```

File: channels/ca.m3u [hunk 9]
```diff
@@ -210,6 +214,8 @@ http://primeasia.dyndns.tv:8080/Live_web_250/index.m3u8
 http://cdn27.live247stream.com/primecanada/247/primecanada/stream1/playlist.m3u8
 #EXTINF:-1 tvg-id="QVTV.ca" tvg-country="CA" tvg-language="English" tvg-logo="https://i.imgur.com/X9LBFzK.jpg" group-title="Religious",QVTV (720p)
 https://qvmstream.tulix.tv/720p/720p/playlist.m3u8
+#EXTINF:-1 tvg-id="SikhSpiritualCentreRexdale.com" tvg-country="CA" tvg-language="Punjabi" tvg-logo="https://www.sikhspiritualcentrerexdale.com/wp-content/uploads/2019/07/favicon-transprent.png" group-title="",Rexdale Gurdwara Sahib
```
Comment (freearhey) [inline channels/ca.m3u:217]:
```suggestion
#EXTINF:-1 tvg-id="RexdaleGurdwaraSahib.ca" tvg-country="CA" tvg-language="Punjabi" tvg-logo="https://www.sikhspiritualcentrerexdale.com/wp-content/uploads/2019/07/favicon-transprent.png" group-title="",Rexdale Gurdwara Sahib
```

Comment (tpkahlon) [reply channels/ca.m3u]: @freearhey Fixed!

File: channels/ca.m3u (hunk 8 above)
Comment (tpkahlon) [reply channels/ca.m3u]: @freearhey Fixed!

File: channels/ca.m3u (hunk 7 above)
Comment (tpkahlon) [reply channels/ca.m3u]: @freearhey Fixed!

File: channels/ca.m3u [hunk 10]
```diff
@@ -165,6 +167,8 @@ https://live.corusdigitaldev.com/groupd/live/deb8cec5-87fb-460a-ab82-0929374fc5f
 https://live.corusdigitaldev.com/groupb/live/564df695-94f9-4f27-b1b9-0a936ab01721/live.isml/.m3u8
 #EXTINF:-1 tvg-id="GurSikhSabhaTV.ca" tvg-country="CA" tvg-language="Hindi" tvg-logo="https://i.imgur.com/2EKCtQm.png" group-title="Religious",GurSikh Sabha TV (720p)
 http://cdn12.henico.net:8080/live/gsctv/index.m3u8
+#EXTINF:-1 tvg-id="RamgarhiABC.com" tvg-country="CA" tvg-language="Punjabi" tvg-logo="http://ramgarhiabc.com/wp-content/uploads/2014/02/headerlogo.png" group-title="",Ramgarhi ABC
```
Comment (freearhey) [inline channels/ca.m3u:170]:
```suggestion
#EXTINF:-1 tvg-id="RamgarhiABC.ca" tvg-country="CA" tvg-language="Punjabi" tvg-logo="http://ramgarhiabc.com/wp-content/uploads/2014/02/headerlogo.png" group-title="",Ramgarhi ABC
```

File: channels/ca.m3u [hunk 11]
```diff
@@ -210,6 +214,8 @@ http://primeasia.dyndns.tv:8080/Live_web_250/index.m3u8
 http://cdn27.live247stream.com/primecanada/247/primecanada/stream1/playlist.m3u8
 #EXTINF:-1 tvg-id="QVTV.ca" tvg-country="CA" tvg-language="English" tvg-logo="https://i.imgur.com/X9LBFzK.jpg" group-title="Religious",QVTV (720p)
 https://qvmstream.tulix.tv/720p/720p/playlist.m3u8
+#EXTINF:-1 tvg-id="SikhSpiritualCentreRexdale.com" tvg-country="CA" tvg-language="Punjabi" tvg-logo="https://www.sikhspiritualcentrerexdale.com/wp-content/uploads/2019/07/favicon-transprent.png" group-title="",Sikh Spiritual Centre Rexdale
```
Comment (freearhey) [inline channels/ca.m3u:217]:
```suggestion
#EXTINF:-1 tvg-id="SikhSpiritualCentreRexdale.ca" tvg-country="CA" tvg-language="Punjabi" tvg-logo="https://www.sikhspiritualcentrerexdale.com/wp-content/uploads/2019/07/favicon-transprent.png" group-title="",Sikh Spiritual Centre Rexdale
```

**Review by freearhey — DISMISSED** [review]
`tvg-id` should consist of the channel name and country code separated by a dot

**Comment by tpkahlon** [comment]
@freearhey Looks like all three new additions has tvg-id value listed with stream name followed by a dot.

File: channels/ca.m3u (hunk 11 above)
Comment (tpkahlon) [reply channels/ca.m3u:217]: @freearhey Can we use site address in the tvg-id? Some channels are located at standard `.com` address and do not end with `.ca`.

File: channels/ca.m3u (hunk 10 above)
Comment (tpkahlon) [reply channels/ca.m3u:170]: @freearhey Can we use site address in the tvg-id? Some channels are located at standard `.com` address and do not end with `.ca`.

File: channels/ca.m3u (hunk 11 above)
Comment (LaneSh4d0w) [reply channels/ca.m3u:185]: tvg-id isn't the address of the website. It's an unique name that permits to identify a channel.

File: channels/ca.m3u [hunk 12]
```diff
@@ -95,6 +95,10 @@ https://d7z3qjdsxbwoq.cloudfront.net/groupa/live/f9809cea-1e07-47cd-a94d-2ddd3e1
 http://159.69.58.154/cpmtv/cpmtv.m3u8
 #EXTINF:-1 tvg-id="CTV.ca" tvg-country="CA" tvg-language="English" tvg-logo="https://i.imgur.com/nfcjHAW.png" group-title="",CTV (720p)
 https://pe-fa-lp02a.9c9media.com/live/News1Digi/p/hls/00000201/38ef78f479b07aa0/index/0c6a10a2/live/stream/h264/v1/3500000/manifest.m3u8
+#EXTINF:-1 tvg-id="CTVToronto.ca" tvg-country="CA" tvg-language="English" tvg-logo="https://d229kpbsb5jevy.cloudfront.net/teleup/content/common/logos/channel/logos/uafvsq.png" group-title="",CTV Toronto (720p)
+https://bozztv.com/teleyupp1/teleup-zxsJFt6VvY/playlist.m3u8
```
Comment (freearhey) [inline channels/ca.m3u:99]: Should be moved to `us_teleup.m3u`

Comment (Dum4G) [reply channels/ca.m3u:99]: @tpkahlon

**Review by Dum4G — APPROVED** [review]

**Pull request merged**